packages/select/src/ListMenu.tsx

```diff
@@ -1,11 +1,15 @@
 import React, { useCallback, useContext } from 'react';
 import { css, cx } from '@leafygreen-ui/emotion';
-import { useViewportSize } from '@leafygreen-ui/hooks';
 import Popover, { Align, Justify, PopoverProps } from '@leafygreen-ui/popover';
 import { breakpoints, fontFamilies } from '@leafygreen-ui/tokens';
+import { useViewportSize } from '@leafygreen-ui/hooks';
 import SelectContext from './SelectContext';
 import { colorSets, mobileSizeSet, sizeSets } from './styleSets';
 import { useForwardedRef } from './utils';
+import { useMemo } from 'react';
+
+const MAX_MENU_HEIGHT = 274;
```

Comment:
Is there a reason to use snake-case instead of camel-case here?

Reply:
None other than I've typically used all caps snake case for config constants like this

Reply:
We use camelcase for sizing-related variables in a number of places already like Code, Menu, and Checkbox.